templates/taxi/driver_detail.html

```diff
@@ -3,6 +3,10 @@
 {% block content %}
   <h1>
     Username: {{ driver.username }}
+    <span class="position-relative pl-5">
+      <a href="{% url 'taxi:driver-update' pk=driver.id %}" class="btn btn-success">Update license</a>
```

Comment:
Ensure that the URL name 'taxi:driver-update' used here matches the patterns defined in `urls.py`. Consistency in naming is crucial for maintaining the functionality across the application.